packages/sync-actions/test/stores-sync.spec.js

```diff
@@ -0,0 +1,51 @@
+import storesSyncFn, { actionGroups } from '../src/stores'
+import { baseActionsList } from '../src/stores-actions'
+
+describe('Exports', () => {
+  test('action group list', () => {
+    expect(actionGroups).toEqual(['base'])
+  })
+
+  test('correctly define base actions list', () => {
+    expect(baseActionsList).toEqual([
+      { action: 'setLanguages', key: 'languages' },
+      { action: 'setName', key: 'name' },
+    ])
+  })
+})
+
+describe('Actions', () => {
+  let storesSync
+  beforeEach(() => {
+    storesSync = storesSyncFn()
+  })
+
+  test('should build `setName` action', () => {
+    const before = {
+      name: { locale: 'en', value: 'Algeria' },
```

Comment:
A `LocalizedString` has the format (see https://docs.commercetools.com/http-api-types.html#localizedstring):
```js
name: {
  'en': 'Algeria'
...
}
```

Reply:
This is true and `sync-actions` only deals with REST shapes so far.
We should change this, @amine-benselim